http/nodeinfo/server.go

```diff
@@ -111,11 +113,24 @@ func viewHandler(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func peerAddressListHandler(w http.ResponseWriter, r *http.Request) {
+	list := make([]string, 0)
+
+	nbrs := node.GetNeighbors()
+	for _, v := range nbrs {
+		list = append(list, v.GetAddr())
```

Comment:
suggest updating PeerInfo.String(), and show all peer info.

Reply:
ok